package.json

```diff
@@ -65,14 +65,13 @@
     "meow": "^10.0.1",
     "mocha": "^9.1.3",
     "nyc": "^15.1.0",
-    "parcel-bundler": "^1.12.5",
+    "parcel": "^2.0.0",
```

Comment:
Parcel is at 2.0.1 right now, so we might as well use that, https://www.npmjs.com/package/parcel